pkg/plugins/resources/postgres/pgx_store.go

```diff
@@ -95,7 +95,7 @@ func (r *pgxResourceStore) Create(ctx context.Context, resource core_model.Resou
 	}
 
 	version := 0
-	statement := `INSERT INTO resources VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11);`
+	statement := `INSERT INTO resources (name, mesh, type, version, spec, creation_time, modification_time, owner_name, owner_mesh, owner_type, labels) VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11);`
```

Comment:
Did it cause any problems or just for visibility?

Reply:
https://kongstrong.slack.com/archives/C04Q8M7BNSU/p1702990501474759?thread_ts=1702981199.008899&cid=C04Q8M7BNSU